lib/kwtool_web/controllers/home_controller.ex

```diff
@@ -0,0 +1,9 @@
+defmodule KwtoolWeb.HomeController do
+  use KwtoolWeb, :controller
+
+  def index(conn, _params) do
+    user = Guardian.Plug.current_resource(conn)
```

Comment:
I think it might be easier if we have another plug to set the current user to conn assigns. 

Then in the code, we can just do `conn.assigns.current_user`

or in the template where we want to show something we can do `<%= if @conn.assigns.current_user do %>`

So we don't need to call Guardian everywhere. This will be easier for us later if we change the authentication lib. 👍

Reply:
Updated to `conn.assigns.current_user`